app/models/order.rb

```diff
@@ -1,10 +1,33 @@
-class Order <ApplicationRecord
+class Order < ApplicationRecord
   validates_presence_of :name, :address, :city, :state, :zip
 
   has_many :item_orders
   has_many :items, through: :item_orders
+  belongs_to :user
```

Comment:
This is not a comment for Shaunda but an odd thing. I had this in my code, but I don't see it in main. Rather weird.